src/main/java/com/icthh/xm/uaa/security/ldap/LdapAuthenticationProviderBuilder.java

```diff
@@ -22,16 +22,20 @@
 import org.springframework.security.ldap.authentication.ad.ActiveDirectoryLdapAuthenticationProvider;
 import org.springframework.security.ldap.userdetails.DefaultLdapAuthoritiesPopulator;
 import org.springframework.stereotype.Component;
+import com.icthh.xm.commons.lep.LogicExtensionPoint;
+import com.icthh.xm.commons.lep.spring.LepService;
 
 @AllArgsConstructor
 @Slf4j
 @Component
+@LepService(group = "security.provider")
 public class LdapAuthenticationProviderBuilder {
 
     private final TenantPropertiesService tenantPropertiesService;
     private final DomainUserDetailsService userDetailsService;
     private final UserService userService;
 
+    @LogicExtensionPoint("BuildLdapProvider")
     public Optional<AuthenticationProvider> build(String domain) {
```

Comment:
it is necessary to add a resolver on the domain

Reply:
done.